libraries/chain_kv/include/b1/chain_kv/chain_kv.hpp

```diff
@@ -110,12 +111,141 @@ inline bytes get_next_prefix(const bytes& prefix) {
    return next_prefix;
 }
 
+namespace detail {
+   typedef __uint128_t uint128_t;
```

Comment:
What's the stance on typedef vs alias at block.one?

Reply:
I had quickly copied it, but I would definitely prefer using, so changing it.  Don't know what other reasons there are, but it is way easier to search and find "using TYPE-NAME =" then having to make a regex to find "typedef ... TYPE-NAME;"